src/neo/Network/P2P/RemoteNode.cs

```diff
@@ -112,7 +116,21 @@ protected override void OnData(ByteString data)
             msg_buffer = msg_buffer.Concat(data);
 
             for (Message message = TryParseMessage(); message != null; message = TryParseMessage())
+            {
+                if (message_count == 0)
+                    start_message_time = TimeProvider.Current.UtcNow;
+                else if (message_count > 1000)
+                {
+                    message_count = 0;
+                    if ((TimeProvider.Current.UtcNow - start_message_time).TotalMilliseconds < time_threshold)
```

Comment:
We have a threshold for receive, but not for send, what happens if it's a normal behaviour be fast?